apps/nextra/pages/en/network/nodes/configure/consensus-observer.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Consensus Observer"
+---
+
+import { Callout, Steps } from 'nextra/components';
+
+# Consensus Observer
+
+<Callout type="warning">
+**Public Fullnodes (PFNs)**<br />
+The configuration settings described in this document are for public fullnodes (PFNs) only.
+Consensus observer has already been enabled (by default) for validators and validator fullnodes (VFNs).
+It is not recommended to change the consensus observer configurations for validators and VFNs.
+</Callout>
+
+Consensus observer is a new data dissemination technique that reduces block synchronization time, and
+transaction latencies for Aptos fullnodes (i.e., VFNs and PFNs). It has been shown to reduce end-to-end
+transaction latencies in Mainnet by: (i) ~10% at low load (e.g., 500 TPS); (ii) ~30% at medium load
+(e.g., 5k TPS); and (iii) ~50%+ at higher loads (e.g., 10k TPS+).
+
+More information on consensus observer can be found in the AIP: [AIP-93: Consensus Observer](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-93.md).
+
+## Configure consensus observer
+
+<Callout type="warning" emoji="❗">
+**Minimum Hardware Requirements**<br />
```

Comment:
You can put two newlines rather than a <br /> to make an easier flow

Reply:
Can you clarify what you mean? Remove the `<br />` and replace it with 2 lines? If so, it looks slightly different and I kinda prefer this one 😄 Also, this is consistent with other callouts in the codebase.

Reply:
Right, you have to put <br> in the callouts